recipes/tt-metalium/meta.yaml

```
@@ -0,0 +1,67 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+# If your package is python based, we recommend using Grayskull to generate it instead:
+# https://github.com/conda-incubator/grayskull
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 16 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "tt-metalium" %}
+{% set version = "0.55.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/tenstorrent/tt-metal/releases/download/v{{ version }}-rc13/tt-metalium.tar.gz
+    sha256: 5b4b4454c6fe60e91730b676ccc126fdc41149738e06f3e8d34bb9979bf30259
+
+build:
+  number: 0
+  skip: true  # [not linux or py!=310]
```

Am curious why it needs to be Python 3.10 only. Is there more context on this?

---

3.8 and 3.10 are the only versions known to work. I was seeing some Python packaging issues and decided to try constraining this. I’ll see if I can open it back up.

---

There's very little variation between python versions that would make this plausible. I'm also for removing the constraint (even if something fails, I expect this to be quite manageable).
```suggestion
  skip: true  # [not linux]
```